### PR TITLE
blockchain: reuse channel variable

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

This pr merges the blockchain channel variables. This is a minor change and more cosmetic than anything else. If people would like to leave it as is, we can.

not sure if this constitutes a changelog entry?

Closes: #XXX

### CODE REVIEW

**Comment by erikgrinaker** [comment]
Personally, I'd like to keep the reactors independent. They're separate implementations.

**Comment by marbar3778** [comment]
> Personally, I'd like to keep the reactors independent. They're separate implementations.

I am fine with this, but this contradicts the sharing of messages as it's currently implemented on master.

File: blockchain/util.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package blockchain
```
Comment (melekes) [inline blockchain/util.go:1]: can we rename the file name to something more meaningful? like `channel.go`

**Comment by marbar3778** [comment]
closing this in favor of keeping the channel distinction seperate for each reactor

**Pull request closed without merging**